src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -164,5 +170,34 @@ internal static Settings FromDefaultSources()
 
             return new Settings(configurationSource);
         }
+
+        private static OtlpExportProtocol? GetExporterOtlpProtocol(IConfigurationSource source)
+        {
+            // the defualt in SDK is grpc. http/protobuf should be default for our purposes
+            var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
+
+            if (string.IsNullOrEmpty(exporterOtlpProtocol) || exporterOtlpProtocol == "http/protobuf")
+            {
+                // override settings only for http/protobuf
+                // the second part of the condition is needed to override default endpoint as long as https://github.com/open-telemetry/opentelemetry-dotnet/pull/2868 is not fixed
+                return OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf;
+            }
+
+            return null;
```

Comment:
Returning `null` here is surprising without the full context of the PR and interaction with the SDK. I think it deserves a comment.

Reply:
Done,